cypress/e2e/supervision/help-and-guidance/help-and-guidance.cy.js

```diff
@@ -1,16 +1,15 @@
 describe("Help and Guidance", { tags: ["@supervision", "@smoke-journey"] }, () => {
-  it("content is accessible when expanded", () => {
-    cy.loginAs("Case Manager");
-    cy.visit("/supervision/#/dashboard");
+    it("content is accessible when expanded", () => {
+      cy.loginAs("Case Manager");
+      cy.visit("/supervision/#/dashboard");
 
-//     cy.window().then((win) => {
-//       cy.stub(win, 'open', url => {
-//         win.location.href = 'https://wordpress.sirius.opg.service.justice.gov.uk/';
-//       }).as("popup")
-//     })
-//     cy.get('#open-help-and-guidance-main-menu-link').click()
-//     cy.get('@popup').should("be.called")
-//     cy.get('h1').should('have.class', "help-header");
-//     cy.get('h1').should('contain.text', "Help and Guidance");
-  })
+      cy.get('#open-help-and-guidance-main-menu-link')
+        .should('be.visible')
+        .then(($a) => {
+          expect($a).to.have.attr('target', '_blank')
+          // update attr to open in same tab
+          $a.attr('target', '_self')
+        })
+        .click()
+    });
```

Comment:
Could it be worth asserting anything after this click, to ensure the button works & page loads?

Reply:
The problem is that the page is different in each environment, on localhost it's literally an empty page without the headers/ css we want. The url is never the same (i.e. I can't check it always contains the word help) so I'm not sure what else to do.

Reply:
I've added an assert that it's changed from the dash board page